pap-frontend/src/app/app.module.ts

```diff
@@ -3,10 +3,14 @@ import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { CreateProductComponent } from './product/page/create-product/create-product.component';
+import { ListProductComponent } from './product/page/list-product/list-product.component';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    CreateProductComponent,
```

Comment:
Create ProductsModule and add angular routing to lazy load the module on route "/products"

Reply:
J'ajoute le routing des composants dans le routing de ProductsModule ?

Reply:
Oui

Reply:
Va sur slack si tu as des questions